programming/sr/cheat_sheet.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: page
+title: API Quick Reference
```

Comment:
Should this be in the navigation too?

Reply:
Definitely, please could you point me in the direction of the file I need to change?

Reply:
`_data/sidebar_tree.yaml`

Reply:
It's in `_data/sidebar_tree.yaml`.

Reply:
d17012b16c2c1fc93f670fd3addbb9b0a72e35dc